app/pods/publications/overview/reports/controller.js

```diff
@@ -1,70 +1,78 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
-import moment from 'moment';
+
+export class ReportTypeEntry {
+  constructor(lastJob, type, config) {
+    this.lastJob = lastJob;
+    this.type = type;
+    this.config = config;
+  }
+}
 
 export default class PublicationsOverviewReportsController extends Controller {
   @service store;
+  @service router;
   @service intl;
   @service currentSession;
   @service toaster;
   @service jobMonitor;
 
-  createExportJob(reportTypeUri) {
+  createExportJob(reportTypeEntry, userParams) {
     const now = new Date();
 
-    // TODO: Aside from a default (which can be constructed as follows),
-    // Configurable naming isn't in the current scope of service functionality and hence shouldn't be part of config.
-    // In case the frontend has special naming requirements, then these can be fullfilled
-    // by editing the file name after the fact in the frontend and saving through mu-cl-resources
-    const reportNameDatePrefix = moment(now).format('YYYYMMDDHHmmss');
-    const reportNameType = 'to-be-moved-to-backend';
-    const reportName = `${reportNameDatePrefix}-${reportNameType}`;
+    const fixedParams = reportTypeEntry.config.fixedParams;
+    /// TOREVIEW: use _.merge?
```

Comment:
I suppose `_.merge` would only have an advantage over `...` in case `fixedParams.query.filter` has more than 1 nested level. I don't think it does, plus eventually this will be moved to the `publication-report-service` anyway, so this is fine the way it is.